archive/frames/serializers.py

```diff
@@ -3,6 +3,7 @@
 from archive.frames.models import Frame, Version, Headers
 from django.contrib.gis.geos import GEOSGeometry
 from django.db import transaction
+from archive.settings import ZIP_DOWNLOAD_MAX_UNCOMPRESSED_FILES
```

Comment:
I think you should import this as `from django.conf import settings` and then use `settings.ZIP_DOWNLOAD_MAX_UNCOMPRESSED_FILES`.